docker/install-deps

```diff
@@ -23,6 +23,12 @@ if [ -f /usr/bin/yum ]; then
     curl https://jenkins.percona.com/downloads/cent6/centos6-eol.repo --output /etc/yum.repos.d/CentOS-Base.repo
   fi
 
+  if [[ ${RHVER} -eq 8 ]]; then
+    yum -y install centos-release-stream
+    dnf -y swap centos-{linux,stream}-repos
+    dnf -y distro-sync
```

Comment:
this is not needed. you can just enable repo and not convert builder to centos stream

Reply:
Fixed